### ISSUE TITLE
Not working on Event.ACTIVATE

### ISSUE THREAD

**Issue by ekeeper** [freshplanet/ANE-Background-Music#2]
When I use AirBackgroundMusic.getInstance().initialize(); in the init method of my app - all is ok, but I'm always want to check audio on ACTIVATE event. So it does not work and iOS audio become mixed with game audio.

**Comment by Pol64** [comment]
Exactly. Please can you just implement the dispose() method. So then I can dispose the BackgroundMusic ANE when app is deactivated and reinitialize when app is activated ? Should help us.

Thanks for very fast update :)

**Comment by Pol64** [comment]
Any response please ?

**Comment by skizzo** [comment]
**+1 for a dispose method**, since
1. calling _initialize()_ only once when launching the app is not an option since the user can suspend/deactivate the app, start playback somewhere else and then return to it, leaving the playback started outside the app still running; and
2. a second call of the _initialize()_ function (e.g. before actually starting playback) leaves any sound objects muted.

Would be really great, thanks!
